apps/wallet-mobile/src/features/ReviewTx/common/CopiableText.tsx

```diff
@@ -1,46 +1,51 @@
 import {useTheme} from '@yoroi/theme'
 import * as React from 'react'
-import {StyleSheet, Text, TextStyle, TouchableOpacity, View} from 'react-native'
+import {StyleSheet, Text, TextProps, TextStyle, TouchableOpacity, View} from 'react-native'
 
 import {Icon} from '../../../components/Icon'
 import {Space} from '../../../components/Space/Space'
 import {useCopy} from '../../../hooks/useCopy'
 
-export const Address = ({
-  address,
+export const CopiableText = ({
+  text,
   index,
   textStyle,
   multiline = false,
+  addressProps,
 }: {
-  address: string
+  text: string
   index?: number
   textStyle?: TextStyle
   multiline?: boolean
+  addressProps?: TextProps
 }) => {
   const {styles, colors} = useStyles()
   const [, copy] = useCopy()
 
   return (
-    <View style={styles.address}>
+    <View style={styles.text}>
       <Text
```

Comment:
if it is requiring too much configuration, receive as children.

Reply:
Yes! After that these abstractions become very complex and difficult to debug. Refactored